truffle-config.js

```diff
@@ -77,7 +77,9 @@ module.exports = {
   },
   compilers: {
     solc: {
-      version: '0.5.11',
+      version: '0.5.17',
+      docker: true,
+      parser: 'solcjs',
```

Comment:
Are we still using solcjs? shouldn't we use sold binary?

Reply:
this is only for parsing, truffle says it is faster than binary.